src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -12,4 +16,5 @@ public Bulldozer() {
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
```

Comment:
The task description requires adding custom fields specific to a bulldozer. Consider adding fields such as blade width, engine power, or any other relevant attributes that are unique to a bulldozer.